routes/authRouter.js

```diff
@@ -0,0 +1,16 @@
+const Router = require('express')
+const router = new Router()
+const authController = require('../controllers/authController')
+const {check} = require('express-validator')
+const authMiddleware = require('../middleware/authMiddleware')
```

Comment:
Let's follow the next imports order:
1) External dependencies 
2) Internal dependencies
3) Constants and other stuff

```suggestion
const Router = require('express');
const { check } = require('express-validator');
const authMiddleware = require('../middleware/authMiddleware');
const authController = require('../controllers/authController');

const router = new Router();
```

Reply:
Also check other places